packages/core/admin/admin/src/components/UnstableGuidedTour/Step.tsx

```diff
@@ -48,6 +50,21 @@ type Step = {
   Actions: (props: ActionsProps & { to?: string }) => React.ReactNode;
 };
 
+const GUIDED_TOUR_TRACKING_EVENTS = {
+  contentTypeBuilder: {
+    create: 'didCreateGuidedTourCollectionType' as EventWithoutProperties['name'],
```

Comment:
I would add `start: 'didClickGuidedTourHomepageContentManager'` and then use the constant in the overview so all tracking is centralized